### ISSUE TITLE
ImageCropping: Manage zooming with mouse wheel

### ISSUE THREAD

**Issue by zaknafein83** [primefaces/primefaces#6175]
**Description**
When i scroll the site, and i go hover imagecrop, then zoom on image, that's frustating

**Describe the solution you would like**
Disable automatic zoom and add control image zoom and positioning like "maps control", with arrows on image to move, plus and minus to zoom.

**Comment by melloware** [comment]
Yep Cropper.js has two properties `zoomOnWheel` and `zoomOnTouch` that can be used.
See: https://github.com/fengyuanchen/cropperjs#zoomonwheel

I will be submitting a PR to add these properties.

**Comment by melloware** [comment]
If you need this fix right now add this JS code to your custom application.js or right on the page.
```javascript
if (PrimeFaces.widget.ImageCropper) {
    PrimeFaces.widget.ImageCropper.prototype.init = function(cfg) {
        // skip calling PrimeFaces.widget.Layout.init() on purpose
        PrimeFaces.widget.DeferredWidget.prototype.init.call(this, cfg);

        this.image = $(PrimeFaces.escapeClientId(this.cfg.image));
        this.jqCoords = $(this.jqId + '_coords');

        // calculate the min and max of the cropper box
        var imageWidth = this.image[0].naturalWidth,
            imageHeight = this.image[0].naturalHeight;
        this.cfg.minCropBoxWidth = this.cfg.minSize ? this.cfg.minSize[0] : 0;
        this.cfg.minCropBoxHeight = this.cfg.minSize ? this.cfg.minSize[1] : 0;
        this.cfg.maxCropBoxWidth = Math.min(imageWidth, this.cfg.maxSize ? this.cfg.maxSize[0] : imageWidth);
        this.cfg.maxCropBoxHeight = Math.min(imageHeight, this.cfg.maxSize ? this.cfg.maxSize[1] : imageHeight);

        this.cfg.data = {
            width: (this.cfg.minCropBoxWidth + this.cfg.maxCropBoxWidth) / 2,
            height: (this.cfg.minCropBoxHeight + this.cfg.maxCropBoxHeight) / 2,
        };

        // prevent zooming
        this.cfg.zoomOnWheel = false;
        this.cfg.zoomOnTouch = false;

        this.renderDeferred();
    }
}
```

**Comment by zaknafein83** [comment]
it works. Thanks

**Comment by melloware** [comment]
No problem.  fixed for good in 9.0
